src/models/models.js

```diff
@@ -0,0 +1,7 @@
+import BookAuthor from "./BookAuthor.js";
+
+export default function associateModels(models) {
+    const { Book, Author } = models;
+    Book.belongsToMany(Author, { through: BookAuthor, as: 'authors' });
+    Author.belongsToMany(Book, { through: BookAuthor, as: 'books' });
+}
```

Comment:
It would be useful to stay near to their Model files for clarity, otherwise might become difficult to see which associations are created for which Models